index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body class="page__body">
+    <header class="main-header">
+        <!--А можно сделать логотип фоном?-->
+        <a class="main-header__logo-link" href="/">
+            <img class="main-header__logo-image" src="image\mobilelogo.svg" width="33,2px" height="36.6px" alt="логотип cat energy"> </a>
+        <!--не уверена в заголовке, может надо было делать H1-->
+        <p class="main-header__title">
+            <b class="main-header__first-word">Cat</b> energy
+        </p>
+        <nav class="main-header__navigation navigtion-mobile">
+            <button class="navigtion-mobile_toogle" type="button">
```

Comment:
Здесь и далее есть ошибки именования классов по БЭМ. Правильность можно проверить в [данном сервисе](https://yoksel.github.io/html-tree/).

`navigtion-mobile_toogle` это класс модификатора блока в стиле Яндекс (одно нижнее подчёркивание), а поскольку по коду видно, что вы используете модификаторы в международном формате (два минуса), то тут банальная опечатка и это класс БЭМ элемента:
```suggestion
            <button class="navigtion-mobile__toogle" type="button">
```